yalla-coop/tempo
# 710 create cookie banner

#### Issues this PR relates to
#710 

how to test : 

there is a storybook for the component

**OR** 

I add it to the app : 
so first you need to be logged in and you will see it at the bottom of the page 
in case you pressed  "I understand" button,  it will hide and add cookies in the browser 
in case you scrolled it will hide and add cookies to the browser
if you want to see it again you need to clear your cookies !!! 

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs
